lib/matplotlib/__init__.py

```diff
@@ -1476,7 +1476,6 @@ def _jupyter_nbextension_paths():
 default_test_modules = [
     'matplotlib.tests.test_png',
     'matplotlib.tests.test_units',
-    'matplotlib.tests.test_widgets',
```

Comment:
The answer to the first question is no as far as our own tests are concerned. But `matplotlib.test()` and `tests.py` will work with pytest in #7974. I have no idea how many people may or may not be using that method, so I don't know whether we should remove it or not.